platform/darwin/src/MGLStyleLayer.h

```diff
@@ -69,6 +69,17 @@ MGL_EXPORT
  */
 @property (nonatomic, assign) float minimumZoomLevel;
 
+#pragma mark Layer Transitions
+
+- (void)setTransition:(MGLTransition)transition forKey:(NSString *)key;
+
+- (MGLTransition)transitionForKey:(NSString *)key;
+
+/**
+ Returns an array of strings that identify transition keys attached to the layer.
+ */
++ (NSArray *)transitionKeys;
```

Comment:
This Method is intended to explicitly return which layer attributes are transitionable, we can as well add a note on each property, but I think this is a better way to let know devs about what they can change, also this can be used as helper to set the same transition property to all transitionable attributes